radiography/src/main/java/radiography/FocusedWindowViewFilter.kt

```diff
@@ -7,7 +7,7 @@ import android.view.View
  */
 object FocusedWindowViewFilter : ViewFilter {
 
-  override fun matches(view: View): Boolean {
-    return view.parent?.parent != null || view.hasWindowFocus()
+  override fun matches(view: Any): Boolean {
```

Comment:
should we use a sealed class instead?

a la:
```
sealed class View {
  class AndroidView: View()
  class Composable: View()
}
```

Reply:
It's not clear what `Composable` would even be yet, so we couldn't write that code at this time.